extensions/terminal-suggest/src/test/completions/cd.test.ts

```diff
@@ -8,20 +8,20 @@ import cdSpec from '../../completions/cd';
 import { testPaths, type ISuiteSpec } from '../helpers';
 
 const expectedCompletions = ['-'];
-
+const cdExpectedCompletions = [{ label: 'cd', description: (cdSpec as any).description }];
```

Comment:
@Tyriar these as any casts were needed bc it said `description` did not exist on `fig.spec`, which makes no sense